src/qibo/models/encodings.py

```diff
@@ -716,6 +874,22 @@ def _get_next_bistring(bitstring, markers, hamming_weight):
 
 
 def _ehrlich_algorithm(initial_string, return_indices: bool = True):
+    """Return list of bitstrings with mininal Hamming distance between consecutive strings.
+
+    Based on the Gray code called Ehrlich algorithm.
+
+    Args:
+        initial_string (_type_): _description_
+        return_indices (bool, optional): _description_. Defaults to True.
+
+    Returns:
+        _type_: _description_
+
```

Comment:
This is what I meant her with uncomplete @renatomello! I would address these placeholders before merging! :)